web/layouts/auth-layout/workspace-wrapper.tsx

```diff
@@ -34,6 +35,8 @@ export const WorkspaceAuthWrapper: FC<IWorkspaceAuthWrapper> = observer((props)
   } = useMember();
   const { workspaces } = useWorkspace();
   const { isMobile } = usePlatformOS();
+  //swr hook for fetching issue properties
+  useWorkspaceIssueProperties(workspaceSlug);
```

Comment:
It will fetch all the data when not needed. please dont use this in workspace wrapper. this will make load times longer.

Reply:
I've implemented the modifications you recommended.